components/brave_news/browser/resources/feed/Card.tsx

```diff
@@ -69,7 +73,14 @@ const HidableImage = ({
     src += `&target_size=${width}x${height}`
   }
 
-  return <img {...rest} src={src} ref={ref as any} onError={handleError} onLoad={handleLoad} />
+  return <img
+    {...rest}
+    src={delayLoad ? undefined : src}
+    ref={ref as any}
+    onError={handleError}
```

Comment:
do we still need this `onError` handler, now the image isn't set to visible until it loads?